### DISCUSSION TITLE
any Time frame on when cozy nest will be updated for automatic1111 v1.5.1

### DISCUSSION THREAD

**Discussion by snagi12** [Nevysha/Cozy-Nest#189] [General]
title says it all but is there any time frame for release or beta for automatic1111 v1.5.1.  i had to update to get better support for SDXL and miss several features of cozy nest.

**Comment by kai-rin** [comment]
I miss Cozy-Next, too.

**Comment by snagi12** [comment]
Sadly I guess I have to drop this from my install.  It looks like we have another Automatic1111 update before we get a fix for 1.5.1 since Automatic1111 is now 1.5.2.  Sad cause this has a lot of potential and was the best UI setup I found for Automatic1111.  If I see an update I might try it again, but I am looking around now for another UI overhaul that is kept up to date.
